src/Raven.Studio/typescript/viewmodels/database/indexes/deleteIndexesConfirm.ts

```diff
@@ -3,35 +3,85 @@ import dialog = require("plugins/dialog");
 import database = require("models/resources/database");
 import dialogViewModelBase = require("viewmodels/dialogViewModelBase");
 import messagePublisher = require("common/messagePublisher");
+import index = require("models/database/index/index");
+import indexDefinition = require("models/database/index/indexDefinition");
 
-class deleteIndexesConfirm extends dialogViewModelBase {
+class indexInfoForDelete {
+    indexName: string;
+    reduceOutputCollection: string;
+    ReferenceCollection: string;
 
-    deleteTask = $.Deferred<boolean>();
-    title: string;
+    readonly referenceCollectionExtension = "/References";
```

Comment:
what's the purpose of this?

Reply:
When creating the index, if you specify a 'pattern',
then the server creates the referencing documents collection with this extension.
You don't get to choose this collection name.

i.e if reduce-results go to **ABC** collection 
then the referencing documents will be in collection: **ABC/References**

I can of course remove the readonly, and paste directly in the constructor.

Reply:
please leave it. It is fine. I was just curious about purpose. Thanks for explanation. :)